.github/workflows/build-windows.yml

```diff
@@ -286,6 +286,24 @@ jobs:
              Write-Host "Tests failed. test_assert exit status: " $LastExitCocde
              $host.SetShouldExit($LastExitCode)
           }
+
+          cmake --build $env:BUILD_BUILDDIRECTORY\tiledb -j --target tests --config $CMakeBuildType -- /verbosity:minimal
+          Get-ChildItem "$env:BUILD_BUILDDIRECTORY\tiledb\tiledb\" -Recurse -Filter unit_*.exe |
```

Comment:
We should not loop over tests out-of-band (outside of the build system) like this. Tests can be run via ctest with label-based selection to choose the appropriate subset for a given CI configuration.

(ctest also allows running tests in parallel, which may be advantageous assuming sufficient fixture isolation)